defaults/linuxrc

```diff
@@ -55,6 +55,7 @@ run busybox --install -s
 CMDLINE=$(cat /proc/cmdline 2>/dev/null)
 for x in ${CMDLINE}
 do
+	x=${x//\"/}
```

Comment:
Ok, I realized there IS a problem here.
If there are arguments with significant double-quotes, e.g. due to embedded whitespace, this would break them.

They are probably also partially broken already, so now is a good time to fix them.